aldryn_newsblog/search_indexes.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.template import RequestContext
+
+from aldryn_search.utils import get_index_base, strip_tags
+
+from aldryn_newsblog.models import Article
+
+
+class ArticleIndex(get_index_base()):
+    haystack_use_for_indexing = getattr(settings, "ALDRYN_NEWSBLOG_SEARCH", True)
+
+    INDEX_TITLE = True
+
+    def get_title(self, obj):
+        return obj.safe_translation_getter('title')
+
+    def get_description(self, obj):
+        return obj.safe_translation_getter('lead_in')
+
+    def get_index_kwargs(self, language):
```

Comment:
I'm not too familiar with parler but seems like you're adding the language filtering twice, once here by providing the language filter and 2nd in the ```get_index_queryset``` where you also filter by ```language_code```, this in hvad would result in no entries. That said, I don't think it's necessary to provide ```get_index_queryset```.